nim/app.py

```diff
@@ -34,13 +39,19 @@ def update_game_state():
 
 @app.route('/new', methods=['GET', 'POST'])
 def new_game():
-    json = loads('{"min": 3, "max": 12, "piles": 3}')
-    state = app.game.new_game(
-        min=json['min'],
-        max=json['max'],
-        piles=json['piles'],
-    )
+    """
+    route for /new so user can start new game
+    Uses JSON from frontend for params in new game
+    """
+    session['default_prefs'] = DEFAULT_GAME
```

Comment:
Whenever someone calls `/new`, you'll overwrite what you had saved in their session by doing this, this rendering the `default_prefs` variable useless. You should remove that line, as well as the `DEFAULT_GAME` variable altogether since you were only using it here.

Reply:
Yeah you're right, I knew that would happen but left it this way because it gave me an easy way to always pass all parameters to new_game. 
Though there are many better ways to do this and this isn't good implementation at all. 
I'll work on this now

Reply:
We shouldn't be defaulting anything in the route. All game logic should be in game.py.